osutil/export_test.go

```diff
@@ -155,3 +155,27 @@ func MockUname(f func(*syscall.Utsname) error) (restore func()) {
 		syscallUname = old
 	}
 }
+
+func MockFindUid(mock func(name string) (uint64, error)) (restore func()) {
```

Comment:
Do we actually use MockFindUid or MockFindGid ?

Reply:
No, I did but then changed some tests and didn't remove these.

Reply:
I reintroduced it in 78f14d4 for user_test.go.